lib/tutorials/en_US/routing.md

```diff
@@ -65,63 +77,127 @@ server.route({
+Alternatively you may pass the same value to `h.response(value)` and return that from the handler. The result of this call is a response object, that can be chained with additional methods to alter the response before it is sent. For example `h.response('created').code(201)` will send a payload of created with an HTTP status code of 201. You may also set headers, content type, content length, send a redirection response, and many other things that are documented in the [API reference](/api#response-toolkit).
 
-## Config
+The `handler` option must return a value, a promise, or throw an error.
+
+## <a name="options" /> Options
 
 Aside from these three basic elements, you may also specify an `options` parameter for each route. This is where you configure things like [validation](/tutorials/validation), [authentication](/tutorials/auth), prerequisites, payload processing, and caching options. More details can be found in the linked tutorials, as well as the [API reference](/api#route-options).
 
-Here we will look at a couple of options designed to help generate documentation.
+Here we will look at some options of validating with Joi.
 
-```javascript
+```js
 server.route({
-    method: 'GET',
-    path: '/hello/{user?}',
-    handler: function (request, h) {
+    method: 'POST',
+    path: '/signup',
+    handler: (request, h) => {
 
-        const user = request.params.user ?
-            encodeURIComponent(request.params.user) :
-            'stranger';
-
-        return `Hello ${user}!`;
+        const payload = request.payload;
+        return `Welcome ${encodeURIComponent(payload.username)}!`;
     },
     options: {
-        description: 'Say hello!',
-        notes: 'The user parameter defaults to \'stranger\' if unspecified',
-        tags: ['api', 'greeting']
+        auth: false,
+        validate: {
+            payload: {
+                username: Joi.string().min(1).max(20),
+                password: Joi.string().min(7)
+            }
+        }
     }
 });
 ```
+The first property under `options` is `auth`.  `auth` will set the authentication configuration for the route.  Since this route is for a new user signing up, we will disable authentication.  
+
+The second property is `validate`.  This allows you to set validate rules for various request components, such as `headers`, `params`, `payload`, and `failAction`.  We use the Joi to validate the `request.payload`.  For more info, please check the [validation tutorial](/tutorials/validation).
+
+## <a name="missing" /> 404 Handling
+
+404 errors will happen whenever your server can't find what was a resource that was requested.  It is best practice to handle these errors the proper way.  This is easy to do in hapi, by just employing a route that will catch everything your other routes will not.  
+
+```js
+server.route({
+    method: '*',
+    path: '/{any*}',
+    handler: (request, h) => {
+
+        return h.view('404').code(404);
```

Comment:
If using the `h.view()` method, the example should be fulling working with the vision plugin loaded.

Reply:
done